src/watchdog/watchdog.c

```diff
@@ -0,0 +1,18 @@
+#include <watchdog/watchdog.h>
+#ifndef F_CPU
+#define F_CPU 8000000UL
+#endif
+
+/*volatile ints can be changed at any time (i.e. during ISR)*/
+volatile int LED_ON = 0;
+volatile int interrupt_count = 0;
+char* wdt_mode_to_string = {"SYS RESET","INTERRUPT","BOTH"};
```

Comment:
This gives a warning (type should actually be `char**`), but it doesn't seem to be used anywhere so it should be removed.